index.html

```diff
@@ -161,32 +169,38 @@ <h2 class="title cards">Multi-Post Stories</h2>
             </li>
         </ul>
 
-        <a href="#" class="button project">See Project</a>
+        <a href="#" class="button project" id="last">See Project</a>
     </div>
+</div>
     </section>
 
     <section id="myself">
 
-        <h2 class="title">
+        <div>
+            <h2 class="title" id="last_h2">
                 About Myself
             </h2>
-            <p class="text">
+            <p class="text" id="last_p">
                 I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
             </p>
-            <p id="color">LET’S CONNECT</p>
+            <p id="color" class="last">LET’S CONNECT</p>
 
-            <ul class="social" id="social">
+            <ul class="social end" id="social">
 
                 <li><a href="#"><img src="images/git.svg" id="github" class="icon"></a></li> 
                 <li><a href="#"><img src="images/hand.svg" id="hand" class="icon"></a></li>
                <li><a href="#"><img src="images/linkedin.svg" id="linkedin" class="icon"></a></li>
                <li><a href="#"><img src="images/twitter.svg" id="twitter" class="icon"></a></li>
                 <li><a href="#"><img src="images/M.svg" id="m" class="icon"></a></li>
-            </ul>
-
+            </ul> 
             <p id="resume">Get my resume</p>
+        </div>
```

Comment:
- [ ] Kindly fix the following.

- Your design
<img width="220" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/245dce3b-a402-4898-a383-d5a3e82d8002">

- Figma design
<img width="460" alt="image" src="https://github.com/migcm06/miguel-portfolio/assets/41163571/f703f1df-2469-4140-af05-dc1baeee7759">

- 1. Kindly increase the font size to match the design.
- 2. Reduce the spacing between the social media icons.
- 3. Reduce the padding of the button.